lib/fs/folding_test.go

```diff
@@ -49,6 +49,14 @@ var caseCases = [][2]string{
 	{"a\xCC\x88", "\xC3\xA4"}, // ä
 }
 
+var benchmarkCases = [][2]string{
+	{"img_202401241010.jpg", "ASCII lowercase"},
+	{"IMG_202401241010.jpg", "ASCII mixedcase"},
+	{"übernahme angebot.xlsx", "Unicode lowercase"},
+	{"Übernahme Angebot.xlsx", "Unicode mixedcase"},
```

Comment:
Given the `firstCaseChange` aborted on, well first case change (actually first upper-case) - could you put a string into the mix with upper-case not at the very beginning, e.g. `202401241010-röntgenbild.TIF` :)